### PR TITLE
Also detect UWP as windows

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

File: SDLnetUDP.c [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #include "SDLnetsys.h"
 #include "SDL_net.h"
 
-#if defined(__WIN32__) || defined(__OS2__)
+#if defined(_WIN32) || defined(__OS2__)
```
Comment (madebr) [inline SDLnetUDP.c:25]: SDL2 defines `__WINRT__` for winrt.
```suggestion
#if defined(__WIN32__) || defined(__WINRT__) || defined(__OS2__)
```

**Comment by sezero** [comment]
Changed merge target branch from main to SDL2: Let's not accept any more patches into main until @icculus's #79 gets in.

With that though, you have a conflict which needs resolving.

**Comment by sezero** [comment]
@icculus, @slouken: is this good?

**Comment by slouken** [comment]
Yup!

**Comment by sezero** [comment]
Alright then, this is in.

**Pull request merged**